phase2/kubeadm/do

```diff
@@ -18,13 +18,14 @@ upgrade_master() {
   update_host_kubeadm ${MASTER}
   validate
 
-  INIT_PARAM=$(execute_host ${MASTER} "sudo cat ${KUBEADM_INIT_PARAM_FILE}")
+  execute_host ${MASTER} "sudo sed -i \"s|^kubernetesVersion: .*|kubernetesVersion: \\\"${KUBERNETES_VERSION}\\\"|\" ${KUBEADM_CONFIG_FILE}"
+
   case "${UPGRADE_METHOD}" in
     "init")
-      execute_host ${MASTER} "sudo kubeadm init --skip-preflight-checks --kubernetes-version ${KUBERNETES_VERSION} ${INIT_PARAM}"
+      execute_host ${MASTER} "sudo kubeadm init --skip-preflight-checks --config ${KUBEADM_CONFIG_FILE}"
       ;;
     "upgrade")
-      execute_host ${MASTER} "sudo kubeadm config upload from-flags ${INIT_PARAM}"
+      execute_host ${MASTER} "sudo kubeadm config upload from-file --config ${KUBEADM_CONFIG_FILE}"
```

Comment:
@luxas Is it needed to upload the config file if the config file was used with kubeadm init? 

If it is not needed, we can probably take out the logic uploading the file.

Reply:
It is needed if the cluster was v1.7.x before upgrade.